azurerm/resource_arm_data_lake_store_test.go

```diff
@@ -0,0 +1,208 @@
+package azurerm
+
+import (
+	"fmt"
+	"net/http"
+	"testing"
+
+	"github.com/hashicorp/terraform/helper/acctest"
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/terraform"
+)
+
+func TestAccAzureRMDataLakeStore_payasyougo(t *testing.T) {
```

Comment:
Could we change this to `TestAccAzureRMDataLakeStore_basic`?

Reply:
changed